src/screens/TreeSubmission/screens/SelectOnMap/SelectOnMap.tsx

```diff
@@ -6,17 +6,21 @@ import {Image, StyleSheet, View} from 'react-native';
 import MapMarking from 'screens/TreeSubmission/components/MapMarking/MapMarking';
 import {RootNavigationProp, Routes} from 'navigation';
 import {SafeAreaView} from 'react-native-safe-area-context';
+import useNetInfoConnected from 'utilities/hooks/useNetInfo';
+import SubmitTreeOfflineWebModal from 'components/SubmitTreeOfflineWebModal/SubmitTreeOfflineWebModal';
 
 interface Props extends RootNavigationProp<Routes.SelectOnMap> {}
 
 function SelectOnMap(props: Props) {
+  const isConnected = useNetInfoConnected();
```

Comment:
Please let's follow this convention:
On the first line of your component, always destruct your component's props.